deploy/olm-catalog/kubefed-operator/4.2/kubefed-operator.v4.2.0.clusterserviceversion.yaml

```diff
@@ -304,75 +304,146 @@ spec:
     quickly get up and running with this powerful concept. Federation is a key
     part of any Hybrid Cloud capability.
 
-    **Important Note**: This operator is purely for prototyping purposes and not
+    **Important Note**: 
+    * This operator is purely for prototyping purposes and not
     supported by any upstream community at this time.
 
-
-    **Important Note**: Currently, while in preview, this operator does not
+    * Currently, while in preview, this operator does not
     support automatic upgrades. You must remove the old version of the operator
     manually before installing a new version.
 
 
-    ## Using KubeFed
-
-    This deploys KubeFed in a namespace-scoped as well as in a cluster-scoped fashion.
-    For example, if you subscribe to this package in namespace `my-namespace`,
-    KubeFed will be deployed to manage objects _only_ in `my-namespace` in
-    the host cluster and target clusters.
-
-    After installing KubeFed instance, you will use the `kubefedctl` command
-    line tool to do two types of configuration:
-      
-    - Configure federation to connect to your target clusters with `kubefedctl join`
-    - Create new federation APIs for the API types you want to federate with `kubefedctl enable`
-
-
-    **Important Note**: To add federation capability to your namespace, you are
-    only required to install kubefed in the hosting cluster; there is no need
-    to install kubefed in the target clusters (and no effect gained from
-    doing so).
-
-
-    **Important Note**: The instructions in this guide refer to the namespace
-    where federation is installed as `<namespace>` in command examples. The
-    upstream user guide uses `kube-federation-system` as the name of this namespace.
+    ## Start using the operator
```

Comment:
From this point the information looks like a duplicate of the other file, is this copied over? Is there a better way to single source, e.g. use a template?

Reply:
Yes, it's duplicated except for few links which need to replaced by openshift docs link.